lambo

```diff
@@ -175,6 +179,8 @@ AUTH=false
 NODE=false
 CODEEDITOR=""
 BROWSER=""
+DB_USERNAME=""
```

Comment:
Please make it root here, as well, in case someone doesn't have a config file. Thanks!

Reply:
Sorry :see_no_evil:, should have changed now